src/client/debug/drive-robot.tsx

```diff
@@ -181,6 +234,24 @@ export function DriveRobot(props: DriveRobotProps) {
             <Button icon="stop" onClick={handleStopMove}>
                 Stop
             </Button>
+            <Slider
+                min={-1}
+                max={1}
```

Comment:
Shouldn't be power from 0 to 1? Current implementation looks like it allows inverted controls, which is kinda weird

Reply:
Nick said it should be -1 to 1. This makes sense since our driving logic uses negative values for things like turning and moving backwards. When driving tank style, this feels natural. Pull both stick back to drive backwards, yadda yadda

Reply:
Exactly, they just represent what's in the power level state. Good for debugging the joysticks, plus it looks cool.